playwright-e2e/tests/dsm/medical-record-request.spec.ts

```diff
@@ -243,14 +247,16 @@ test.describe.serial('Medical records request workflow', () => {
 
       // Check the "No Action Needed" checkbox and reload the page. Revisit the same participant, latest action is saved accordingly - The "No Action Needed" checkbox is checked.
       medicalRecordsRequestPage = await medicalRecordTable.openRequestPageByRowIndex(foundRow);
-      await medicalRecordsRequestPage.getNoActionNeeded.check();
+      await medicalRecordsRequestPage.getNoActionNeeded.check({ callback: async () => await waitForResponse(page, { uri: '/patch' }) });
```

Comment:
Need to wait for triggered `/patch` request to finish before next line. One-liner to replace 
```
await Promise.all([
  waitForResponse(page, { uri: '/patch' }) }),
  medicalRecordsRequestPage.getNoActionNeeded.check()
]);

```